src/rgw/rgw_rados.h

```diff
@@ -2873,6 +2873,9 @@ class RGWRados : public AdminSocketHook
       int write_meta(uint64_t size, uint64_t accounted_size,
                      map<std::string, bufferlist>& attrs);
       int write_data(const char *data, uint64_t ofs, uint64_t len, bool exclusive);
+      const req_state* get_req_state() {
+        return (const req_state*)(target->get_ctx().s);
```

Comment:
don't need the cast anymore - and the function itself can be const

Reply:
ok